src/modules/cmdpal/Microsoft.CmdPal.UI/Microsoft.CmdPal.UI.csproj

```diff
@@ -39,9 +39,7 @@
   </PropertyGroup>
 
   <!-- BODGY: XES Versioning and WinAppSDK get into a fight about the app manifest, which breaks WinAppSDK. -->
-  <Target Name="RearrangeXefVersioningAndWinAppSDKResourceGeneration"
-          DependsOnTargets="GetNewAppManifestValues;CreateWinRTRegistration"
-          BeforeTargets="XesWriteVersionInfoResourceFile">
+  <Target Name="RearrangeXefVersioningAndWinAppSDKResourceGeneration" DependsOnTargets="GetNewAppManifestValues;CreateWinRTRegistration" BeforeTargets="XesWriteVersionInfoResourceFile">
```

Comment:
weird that this got reformatted?

Reply:
That is weird. I wonder if the XAML formatter PS script modified it?

Reply:
Ah, this could be my doing. I write project files in vim, and I rarely let VS format them. When VS makes any changes, even incidental ones, it reformats the whole thing.